cli/src/cli/index.ts

```diff
@@ -157,12 +157,11 @@ export const runCli = async () => {
 
   // Explained below why this is in a try/catch block
   try {
-    if (
-      process.env.SHELL?.toLowerCase().includes("git") &&
-      process.env.SHELL?.includes("bash")
-    ) {
-      logger.warn(`  WARNING: It looks like you are using Git Bash which is non-interactive. Please run create-t3-app with another
+    if (process.env.TERM_PROGRAM?.toLowerCase().includes("mintty")) {
+      logger.warn(`  WARNING: It looks like you are using Mintty which is non-interactive. Please run create-t3-app with another
```

Comment:
Could this just be one warning, like `WARNING: It looks like you are using MinTTY, which is non-interactive. This is most likely because you are using Git Bash. In that case, please use Git Bash from another terminal, such as Windows Terminal or the terminal built into Visual Studio Code.`

Reply:
Sure. That message is much better. I’ll make that change once I get home in a few hours.

Reply:
`WARNING: It looks like you are using MinTTY, which is non-interactive. This is most likely because you are using Git Bash. If that's that case, please use Git Bash from another terminal, such as Windows Terminal. Alternatively, you can provide the arguments from the CLI directly: https://create.t3.gg/en/installation#experimental-usage` to skip the prompts.`

Maybe?

Reply:
I like julius's solution because it provied a direct solution as well.

Reply:
Changed the message to the one Julius provided.